### PR TITLE
Add new virtual query_create_table column into system.tables

### PR DESCRIPTION
**Pull request by ludv1x**

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
Let's rename `query_create_table` to `create_table_query` to be more uniform with `SHOW CREATE TABLE`.

Your own test failed in Travis.

File: dbms/src/Databases/DatabaseOrdinary.cpp [hunk 1]
```diff
@@ -403,10 +405,22 @@ time_t DatabaseOrdinary::getTableMetadataModificationTime(
...
+    const Context & context,
     const String & table_name) const
 {
-    ASTPtr ast = getCreateQueryImpl(metadata_path, table_name);
+    ASTPtr ast;
+    try
+    {
+        ast = getCreateQueryImpl(metadata_path, table_name);
+    }
+    catch (Exception & e)
```
Comment (alexey-milovidov) [inline dbms/src/Databases/DatabaseOrdinary.cpp]: const

**Comment by alexey-milovidov** [comment]
Almost Ok.

**Comment by zhang2014** [comment]
Maybe I can rely on this refactor `InterpreterShowCreateQuery`?

**Comment by alexey-milovidov** [comment]
`InterpreterShowCreateQuery` is very simple - I think, nothing to refactor.
And behaviour is slightly different - for `SHOW CREATE TABLE` it is intended to throw an exception if database is unaware of it, but in `system.tables` table, empty string is used in that cases.

**Comment by ludv1x** [comment]
@zhang2014 

> Maybe I can rely on this refactor InterpreterShowCreateQuery?

Did you mean to merge your PR #1824 with this one?

**Comment by zhang2014** [comment]
@ludv1x I mean, rely on this refactor `InterpreterShowCreateQuery` , like `InterpreterShowTablesQuery`. But, as @alexey-milovidov says , their behavior is not consistent

**Pull request merged**